com.unity.ml-agents.extensions/Runtime/Match3/Match3Sensor.cs

```diff
@@ -246,6 +257,15 @@ static void DestroyTexture(Texture2D texture)
                 Object.Destroy(texture);
             }
         }
+
+        public void Dispose()
+        {
+            if (!ReferenceEquals(null, m_ObservationTexture))
+            {
+                DestroyTexture(m_ObservationTexture);
```

Comment:
Is there a place to put sensor utilities and maybe we could move it there? I'm going to add the same thing to GridSensor and that will be the third time we define this method.

Reply:
Let's just put it in Utilities.cs

Reply:
4th time, actually 😄 - Match3, Camera, and RenderTexture sensors already have it

Reply:
Moved.